package.json

```
@@ -71,7 +71,7 @@
     "ts-node": "9.1.1",
     "tsify": "^5.0.2",
     "tslint": "6.1.3",
-    "typescript": "4.1.3",
+    "typescript": "^4.2.4",
```

Typescript creates false minor versions, we should ensure we are using the right version
```suggestion
    "typescript": "4.2.4",
```

---

agree